sabnzbd/filesystem.py

```diff
@@ -166,7 +167,26 @@ def sanitize_filename(name: str) -> str:
     if not name:
         name = "unknown"
 
+    # now split name into name, ext
     name, ext = os.path.splitext(name)
+
+    # If filename is too long, brute-force truncate it:
+    maxlength = DEF_FILE_MAX
+    if len(name) + len(ext) > maxlength:
```

Comment:
No need to introduce extra variable, just use DEF_FILE_MAX everywhere.

Reply:
Done